api/src/main/java/com/grappus/android/api/cache/AuthStore.kt

```diff
@@ -52,4 +61,8 @@ class AuthStore @Inject constructor(
     private fun User?.isNotNull(): Boolean {
         return !authToken.isNullOrEmpty() && user != null && this?.userId != 0L
     }
+
+    internal object Keys {
+        const val AUTH_USER_ID = "auth_user_id"
```

Comment:
Don't we have similar entries in `BaseStore`?

Reply:
Yes, added field there it self.